torchvision/models/quantization/googlenet.py

```diff
@@ -140,7 +148,7 @@ def __init__(self, *args, **kwargs):
         self.quant = torch.quantization.QuantStub()
         self.dequant = torch.quantization.DeQuantStub()
 
-    def forward(self, x):
+    def forward(self, x: Tensor):
```

Comment:
Unsure about return type here.

Reply:
it's `GoogLeNetOutputs` cf. eager_outputs method return type